fdb-extensions/src/main/java/com/apple/foundationdb/async/RangeSet.java

```diff
@@ -84,6 +85,14 @@ private static void checkRange(@Nonnull byte[] begin, @Nonnull byte[] end) {
         }
     }
 
+    public static boolean isFirstKey(@Nonnull byte[] key) {
```

Comment:
`RangeSetTest` should be updated for the new APIs.

Reply:
Added `isFirstOrFinalKeys` test.

Reply:
Have you pushed the changes out?

Reply:
Yep